spec/requests/api/v1/moves_controller_update_spec.rb

```diff
@@ -162,6 +162,34 @@
               response_json.dig('data', 'relationships', 'documents', 'data').map { |document| document['id'] },
             ).to eq(after_documents.pluck(:id))
           end
+
+          context 'when there are no attributes in the params' do
+            let(:move_params) do
+              documents = after_documents.map { |d| { id: d.id, type: 'documents' } }
+              {
+                type: 'moves',
+                relationships: { documents: { data: documents } },
+              }
+            end
+
+            it 'updates the moves documents' do
+              expect(move.reload.documents).to eq(before_documents)
+              do_patch
+              expect(move.reload.documents).to eq(after_documents)
+            end
+
+            it 'does not affect other relationships' do
+              expect { do_patch }.not_to change { move.reload.from_location }
+            end
+
+            it 'returns the updated documents in the response body' do
+              do_patch
+
+              expect(
+                response_json.dig('data', 'relationships', 'documents', 'data').map { |document| document['id'] },
+              ).to eq(after_documents.pluck(:id))
```

Comment:
if order is not an issue here could we use `match_array` to avoid the risk of flakiness?

Reply:
So this is all very weird. It makes me think we're always relying on the natural ordering in the db which is a bit temperamental. We should fix that separately.

Reply:
i've fixed this in my (unrelated) pr - I'd just changed `eq` to `match_array` as we can't assume an order from the database